### PR TITLE
Update apiary.apib

### PR DESCRIPTION
**Pull request by hakob9315**
@hakob9315 Please name with corresponding story number. In this case, I suppose it should be RA-1209, right ?

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -106,4 +106,41 @@ Update referral system for specific organization
         
 + Response 403 (application/json)        
 
-        {status:0 , msg:"Only organization administrators can update referrals."}           
+        {status:0 , msg:"Only organization administrators can update referrals."}        
+        
+### Get referral system subscribers [/organizations/1/referral-system/] [GET]
```
Comment (DimaUlyanets) [inline apiary.apib]: @hakob9315  This url isn't correct, please be more attentive.

File: apiary.apib [hunk 2]
```diff
@@ -106,4 +106,41 @@ Update referral system for specific organization
         
 + Response 403 (application/json)        
 
-        {status:0 , msg:"Only organization administrators can update referrals."}           
+        {status:0 , msg:"Only organization administrators can update referrals."}        
+        
+### Get referral system subscribers [/organizations/1/referral-system/] [GET]        
+
++ Response 200 (application/json)
```
Comment (Joran182to) [inline apiary.apib]: @hakob9315 here we have `subscriber_date`, `created_at`, `updated_at`. I need a date of registration and date of subscription for 'confirmed' referrals and date of registration only for 'potential' referrals. Please describe what is what.
Also, this response should contain `user_name` and `is_new`.
`is_new` - boolean. It should let me know if user already seen this referral. So, if specific referral was not included in this response yet then user doesn't know about him and this referral should have `is_new`: `true`.

File: apiary.apib [hunk 3]
```diff
@@ -106,4 +106,41 @@ Update referral system for specific organization
...
+        {
+           "Potential": [
```
Comment (Joran182to) [inline apiary.apib]: @hakob9315 let's keep it in solid style, so rename it please `Potential` -> `potential` . Same for `Confirmed` -> `confirmed`

Comment (hakob9315) [reply apiary.apib]: ok

File: apiary.apib (hunk 2 above)
Comment (hakob9315) [reply apiary.apib]: created_at  date of registration
subscriber_date  date of subscription

**Pull request merged**